src/main/java/com/auctionapp/api/model/dto/BidDto.java

```diff
@@ -0,0 +1,52 @@
+package com.auctionapp.api.model.dto;
+
+import java.sql.Timestamp;
+import java.util.UUID;
+
+public class BidDto {
+	private UUID id;
+	private Double bidAmount;
+	private Timestamp bidDate;
+	private UserDto buyer;
+	private AuctionDto auction;
+
+	public BidDto() {
+	}
+
+	public BidDto(final UUID id,
+				  final Double bidAmount,
+				  final Timestamp bidDate,
+				  final UserDto buyer,
+				  final AuctionDto auction) {
+
+		this.id = id;
+		this.bidAmount = bidAmount;
+		this.bidDate = bidDate;
+		this.buyer = buyer;
+		this.auction = auction;
+	}
+
+	public UUID getId() {
+		return id;
+	}
+
+	public Double getBidAmount() {
+		return bidAmount;
+	}
+
+	public Timestamp getBidDate() {
+		return bidDate;
+	}
+
+	public void setBidDate(final Timestamp bidDate) {
+		this.bidDate = bidDate;
+	}
```

Comment:
Why do you only have this one setter in this file? Either it shouldn't exist and fields should be final, or all setters should exist.

Reply:
I needed it to set the bid date when creating the bid, but now I deleted it and added default current_timestamp to bidDate column in bid migration file